docs/reference/esql/esql-version.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[esql-version]]
+=== {esql} language versions
+
+++++
+<titleabbrev>Language versions</titleabbrev>
+++++
+
+[discrete]
+[[esql-versions-released]]
+==== Released versions
+
+{esql} is released at version `2024.04.01.🚀`.
+This is the only available version for now.
+
+[discrete]
+[[esql-versions-explanation]]
+=== How versions work
+
+{esql} versions are separate from Elasticsearch versions.
+Their goal is to make sure that queries that were once written
+remain valid even as Elasticsearch and {esql} evolve.
+
+For instance, the following query
```

Comment:
It doesn't read well to have a code sample mid-sentence, so I'd recommend moving lines 41 + 42 up here.